pkg/transport/proxy/httpsse/http_proxy.go

```diff
@@ -98,7 +98,17 @@ func (p *HTTPSSEProxy) Start(_ context.Context) error {
 	mux := http.NewServeMux()
 
 	// Add handlers for SSE and JSON-RPC with middlewares
-	mux.Handle(ssecommon.HTTPSSEEndpoint, applyMiddlewares(http.HandlerFunc(p.handleSSEConnection), p.middlewares...))
+	mux.Handle(ssecommon.HTTPSSEEndpoint, applyMiddlewares(
```

Comment:
Let's add a comment here. At some point we probably want to support streaming HTTP and we'll need to add it here.